BraintreeDropIn/BTCardFormViewController.m

```diff
@@ -472,7 +472,7 @@ - (void)updateSubmitButton {
 
 - (void)advanceFocusFromField:(BTUIKFormField *)currentField {
     NSUInteger currentIdx = [self.requiredFields indexOfObject:currentField];
-    if (currentIdx < [self.requiredFields count] - 1) {
+    if (currentIdx != NSNotFound && currentIdx < self.requiredFields.count - 1) {
         [[self.requiredFields objectAtIndex:currentIdx + 1] becomeFirstResponder];
```

Comment:
The crash was occurring on this line. If the configuration hasn't finished loading, `self.requiredFields` will be an empty array, which means we'll crash when we try to access the object at index `currentIdx +1`.